k8s/yaml/linkis-cg-engineconnmanager.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: linkis-cg-engineconnmanager
+  namespace: preprod
+  labels:
+    app: linkis-cg-engineconnmanager
+    version: v1
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: linkis-cg-engineconnmanager
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: linkis-cg-engineconnmanager
+        version: v1
+    spec:
+      containers:
+        - image: registry.mydomain.com/library/linkis-cg-engineconnmanager:1.2.0
+          imagePullPolicy: Always
+          name: linkis-cg-engineconnmanager
+          ports:
+            - containerPort: 9106
+          lifecycle:
+            postStart:
+              exec:
+                command:
+                  - "sh"
+                  - "-c"
+                  - >
+                    \cp /etc/security/keytabs/krb5.conf /etc/krb5.conf;
+                    \cp /etc/security/keytabs/sssd.conf /etc/sssd/sssd.conf;
+                    sssd;
+                    echo "* */1 * * * /etc/security/keytabs/kinit.sh" >> /var/spool/cron/root;
+                    crond;
+          env:
+            - name: EUREKA_URL
+              value: "http://linkis-mg-eureka:20303/eureka/"
+          volumeMounts:
+            - name: log
+              mountPath: /opt/linkis/linkis-cg-engineconnmanager/logs
+            - name: conf
+              mountPath: /opt/linkis/linkis-cg-engineconnmanager/conf/
+            - name: kerberos
+              mountPath: /etc/security/keytabs/
+              readOnly: true
+            - name: hadoop-config
+              mountPath: /usr/local/hadoop/etc/hadoop
+            - name: hive-config
+              mountPath: /usr/local/hive/conf
+            - name: spark-config
+              mountPath: /usr/local/spark/conf
+      volumes:
+        - name: log
+          hostPath:
```

Comment:
I thinks hostPath is not a good idea. The fixed path would failed the deployment for users.